code/modules/shuttle/marine_dropship.dm

```diff
@@ -215,8 +221,22 @@
 	if(crashing)
 		force = TRUE
 
+	if(automatic_cycle_on && destination == new_dock)
+		if(cycle_timer)
+			deltimer(cycle_timer)
+		cycle_timer = addtimer(CALLBACK(src, .proc/prepare_going_to_previous_destination), rechargeTime + time_between_cycle SECONDS - 20 SECONDS, TIMER_STOPPABLE)
+
 	return ..()
 
+///Announce that the dropship will departure soon
+/obj/docking_port/mobile/marine_dropship/proc/prepare_going_to_previous_destination()
+	cycle_timer = addtimer(CALLBACK(src, .proc/go_to_previous_destination), 20 SECONDS, TIMER_STOPPABLE)
+	priority_announce("Dropship taking off in 20 seconds towards [previous.name]", "Dropship Automatic Departure")
```

Comment:
if timer is say 0 seconds would this message not happen very often

Reply:
Every like three minutes